project-archival.md

```diff
@@ -0,0 +1,41 @@
+# FINOS Project Archiving Process v1.0
```

Comment:
How does this fit with the existing FINOS documentation on archiving projects?

Reply:
I'm having trouble tracking that down! Do you have a link handy?

Reply:
Its at https://community.finos.org/docs/governance/software-projects/stages/archived/.  There's a section on project governance which covers how things currently work

Reply:
Thank you! I've just removed this page and updated the links to point to the community site.